ecommerce/static/js/views/course_detail_view.js

```diff
@@ -68,7 +71,9 @@ define([
                             seat.set('credit_provider_display_name', creditProvider.get('display_name'));
                         }
                     });
-                    html += _.template(CourseCreditSeatsTemplate)({creditSeats: seats.filtered, moment: moment});
+                    html += _.template(CourseCreditSeatsTemplate)({
+                        creditSeats: seats.filtered, moment: moment, currency: this.currency
```

Comment:
If you have two or more properties in the object, you should write them one under one

Reply:
Ok. I fixed it.